models/ac_generator.h

```diff
@@ -97,6 +112,8 @@ class ac_generator : public Node
     double phi_deg_; //!< Phase of sine current (0-360 deg)
 
     Parameters_(); //!< Sets default parameter values
+    Parameters_( const Parameters_& );
+    Parameters_& operator=( const Parameters_& p ); // Copy constructor EN
```

Comment:
Removing this comment from sinusoidal_poisson_generator and sinusoidal_gamma_generator as well.